packages/server/README.md

```diff
@@ -17,72 +17,379 @@ All the provided endpoints are `Content-Type: application/json`
-- Response: 
-  - Code: 200
-  - Content example: 
-    ```json
-        {
-          "exists": true
-        }
-    ```
+  Successful response: 
 
-##### 1.3. All
+  - Code: `200 OK`
+  - Body:
 
-- URL: /users
-- Method: GET
-- Header:
-  - Cookie: `aragonCourtSessionID=<SID>`
-- Query: 
-  - `limit`: Number of items to be fetched
-  - `page`: Page number to be used for the items to be fetched based on the limit requested
-- Response: 
-  - Code: 200
-  - Content example: 
-    ```json
-      {
-        "users":[
-          {
-            "id": 9,
-            "email": "someone@aragon.one",
-            "address": "0x4ecc4fe717d70abee26e7e524b2e6caf29b6217d",
-            "createdAt": "2019-12-25T14:58:58.705Z",
-            "updatedAt":"2019-12-25T14:58:58.705Z"
-          },
-          {
-            "id": 8,
-            "email": "anotherone@aragon.one",
-            "address": "0x4ecc4fe717d70abee26e7e524b2e6caf29b6217d",
-            "createdAt": "2019-12-25T14:58:52.433Z",
-            "updatedAt": "2019-12-25T14:58:52.433Z"
-          }
-        ],
-        "total": 9
-      }
-    ```
+  ```
+  {
+    "emailExists": true || false,
+    "emailVerified": true || false,
+    "addressVerified": true || false,
+    "notificationsDisabled": true || false
+  }
+  ```
+
+##### 1.2. Create session
+
+  This request will authenticate the session with address signature and respond with the appropriate session `Set-Cookie` header. In addition, it will set `"addressVerified": true` using the provided signature.
+
+  All following requests should have the returned `Cookie` HTTP header in order to be authenticated.
+
+  Request:
+
+  - Method: `POST`
+  - Path: `/users/<address>/sessions`
+  - Body:
+
+  ```json
+  {
+    "signature": "0xa8afa92...8b1e52a",
+    "timestamp": 123456
+  }
+  ```
+
+  Successful response:
+
+  - Code: `200 OK`
+  - Body:
+
+  ```json
+  {
+    "authenticated": true
+  }
+  ```
+
+  Error responses: invalid signature / timestamp
+
+  - Code: `400 Bad Request`
+  - Body:
+
+  ```
+  {
+    "errors": [
+      { "signature": "Given signature is invalid" }
+      ...
+      { "timestamp": "Given timestamp is invalid" }
+    ]
+  }
+  ```
+
+##### 1.3. Delete session
+
+  Request:
+
+  - Method: `DELETE`
+  - Path: `/users/<address>/sessions:current`
+  - Body: None
+
+  Successful response:
+
+  - Code: `200 OK`
+  - Body:
+
+  ```json
+  {
+    "deleted": true
+  }
+  ```
+
+##### 1.4. Delete sessions on all user devices
+
+  Request:
+
+  - Method: `DELETE`
+  - Path: `/users/<address>/sessions`
+  - Body: None
+
+  Successful response:
+
+  - Code: `200 OK`
+  - Body:
+
+  ```json
+  {
+    "deleted": true
+  }
+  ```
+
+##### 1.5. Get current email
+
+  Request:
+
+  - Method: `GET`
+  - Path: `/users/<address>/email`
+  - Body: None
+
+  Successful response:
+
+  - Code: `200 OK`
+  - Body:
+
+  ```
+  {
+    "email": "new-juror@aragoncourt.com" || null
+  }
+  ```
+
+##### 1.6. Subscribe juror / Change juror email
```

Comment:
I think the subscription part is not documented below

Reply:
It's the same PUT request, I just copied this from notion